test/e2e/build_test.go

```diff
@@ -41,6 +41,30 @@ var _ = Describe("Podman build", func() {
 		Expect(session).Should(ExitCleanly())
 	})
 
+	It("podman image prune should clean build cache", func() {
+		// try writing something to persistent cache
+		session := podmanTest.Podman([]string{"build", "-f", "build/buildkit-mount/Containerfilecachewrite"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+
+		// try reading something from persistent cache
+		session = podmanTest.Podman([]string{"build", "-f", "build/buildkit-mount/Containerfilecacheread"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+		Expect(session.OutputToString()).To(ContainSubstring("hello"))
+
+		// Prune build cache
+		session = podmanTest.Podman([]string{"image", "prune", "-f", "--build-cache"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+
+		// try reading something from persistent cache should fail
+		session = podmanTest.Podman([]string{"build", "-f", "build/buildkit-mount/Containerfilecacheread"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ErrorToString()).To(ContainSubstring("No such file or directory"))
```

Comment:
this check is redundant as you check the same message below

And in general `No such file or directory` is a way to generic message to check here, that could easily be thrown by podman before ever attempting to build so it makes broken tests very likely so please make sure the error contains the actual path as well

Reply:
Thanks, I will amend this today.